Sources/swift-doc/Subcommands/Generate.swift

```diff
@@ -150,9 +150,9 @@ extension SwiftDoc {
         }
 
         if case .html = format {
-          let cssData = try fetchRemoteCSS()
-          let cssURL = outputDirectoryURL.appendingPathComponent("all.css")
-          try writeFile(cssData, to: cssURL)
+            let cssData = css.data(using: .utf8)!
+            let cssURL = outputDirectoryURL.appendingPathComponent("all.css")
+            try writeFile(cssData, to: cssURL)
```

Comment:
nit: 4 spaces indentation looks inconsistent here with the rest of the file

Reply:
Thanks for catching that. All the more reason to integrate a linter / formatter as discussed in https://github.com/SwiftDocOrg/swift-doc/pull/287#discussion_r637529830.